cmd/deploy.go

```diff
@@ -474,3 +477,36 @@ running 'inertia [REMOTE] init'`,
 		}
 	},
 }
+
+var cmdDeploymentRemove = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove Inertia and shutdown the daemon in the remote VPS",
+	Long: `Remove Inertia directory (~/inertia) and takes down the
+	daemon image from the VPS.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		println("WARNING: This will remove Inertia from the remote")
+		println("as well as take the daemon and is irreversible. Continue? (y/n)")
+		var response string
+		_, err := fmt.Scanln(&response)
+		if err != nil || response != "y" {
+			log.Fatal("aborting")
+		}
+
+		// Daemon down
+		remoteName := strings.Split(cmd.Parent().Use, " ")[0]
+		deployment, _, err := local.GetClient(remoteName, configFilePath, cmd)
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = deployment.DaemonDown()
```

Comment:
maybe add progress output ie `Stopping daemon...` and `Removing Inertia directories...`